nservicebus/azure-service-bus/index.md

```diff
@@ -11,37 +11,37 @@ related:
  - samples/azure/azure-service-bus
 ---
 
-In some environments it is not possible or recommended to rely heavily on distributed transactions to ensure reliability and consistency. Therefore in environments such as very large cloud networks or hybrid networks using MSMQ is not the best idea. In those scenarios a good alternative is Azure Service Bus.
+[Azure Service Bus (ASB)](https://azure.microsoft.com/en-us/services/service-bus/)  is a messaging service hosted on the Azure platform, that allows for exchanging messages between various applications in a loosely coupled fashion. ASB Queues offer FIFO (First In, First Out) guaranteed message delivery, and support a range of standard protocols (REST, AMQP, WS*) and APIs (to put/pull messages on/off the queue). ASB Topics deliver messages to multiple subscribers and allow to use fan-out pattern to deliver messages to downstream systems.
```

Comment:
I would recommend to drop FIFO from the doco. It's not possible with ASB Transport, just with raw ASB.